### ISSUE TITLE
essence.data

### ISSUE THREAD

**Issue by AlucardNosferatu** [schmidtgit/AbathurBot#2]
While I am about to start SC2, the exception reports that the program cannot find 'essence.data' in the bin-dir, I am wondering about how I can get this file?

**Comment by schmidtgit** [comment]
The file should be generated when running the program for the first time.

Having multiple versions of Starcraft in the Base64 is a known course for the essence file not being created.

**Comment by AlucardNosferatu** [comment]
Should I directly run the program to load data from the waken SC2 process, or Should I run SC2 first?

**Comment by schmidtgit** [comment]
Running AbathurBot should automatically launch SC2, if you are using Windows with the game installed at the default location. This does however require that only one version exist in the base64 folder.

**Comment by AlucardNosferatu** [comment]
> Running AbathurBot should automatically launch SC2, if you are using Windows with the game installed at the default location. This does however require that only one version exist in the base64 folder.

Thx for your fast reply.
I have found a dir in my Version directory, its name is Base74071, and there are 2 exe files named "SC2.exe" and "SC2_x64.exe" in it, do you mean the right dir name should be "Base64" instead of "Base74071" ?

**Comment by schmidtgit** [comment]
No, the name is fine.
It will look for a version in C:\Program Files (x86)\StarCraft II\Versions\Base*\SC2.exe.
As long as there is only one "Base____" folder, everything is fine.

Can you verify that SC2 is installed in Program Files (x84)?
If not, change the path in the settings json file that it generates.

**Comment by AlucardNosferatu** [comment]
> No, the name is fine.
> It will look for a version in C:\Program Files (x86)\StarCraft II\Versions\Base*\SC2.exe.
> As long as there is only one "Base____" folder, everything is fine.
> 
> Can you verify that SC2 is installed in Program Files (x84)?
> If not, change the path in the settings json file that it generates.

![image](https://user-images.githubusercontent.com/10172485/58746957-09bfbe80-8497-11e9-93ca-117e8b62b1c2.png)
![image](https://user-images.githubusercontent.com/10172485/58746966-18a67100-8497-11e9-8764-427868bfcb9b.png)

Thx for your patience, looking forward to your reply.

**Comment by AlucardNosferatu** [comment]
I found a suspcious problem, the default map is set to be Cloud Kingdom LE, but the Maps dir do not contain any possiblly corresponding map files:
![image](https://user-images.githubusercontent.com/10172485/58747077-99b23800-8498-11e9-8b44-112257d8f1f2.png)
should I change the map using in AbathurBot?
Thx

**Comment by schmidtgit** [comment]
Yes, try changing the launching map. It should try and do this by itself, but might fail if you have never used the ai version before.
